auth/src/main/java/com/firebase/ui/auth/ui/accountlink/WelcomeBackIdpPrompt.java

```diff
@@ -127,83 +123,71 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     @Override
-    public void onClick(View view) {
-        next(mPrevIdpResponse);
```

Comment:
Yeah it took me a little while to see what was going on here too.  Good find!  This stray listener could have bitten us down the road.